src/Header.js

```diff
@@ -2,7 +2,7 @@ import PropTypes from 'prop-types'
 import React from 'react'
 
 const Header = ({ label }) => {
```

Comment:
I am confused. Is this not handled with a custom header component?

Reply:
Thanks, now I feel like an idiot. Can we pretend this PR never existed? 😅

Reply:
Move along, nothing to see here :)

Reply:
@arecvlohe So, you were right about the label, but I just realized now that wasn't for resource headers. It was for the date headers.

I can create another PR so that resource headers can use a customer header.